lib/loader/index.js

```diff
@@ -47,6 +57,27 @@ function validateArguments (url, options, callback) {
 
 module.exports = {
   /**
+   * Clears the loaded URLs
+   *
+   * Clears any URLs that might have been stored in the loadedUrls hash, either
+   * locally or in the given namespace(optional). This is needed for testing so
```

Comment:
If the only reason this exists is for testing, it probably doesn't need to exist, at least not in the module. Is there another way to get a handle to an internal data structure and clear it via tests? You can get a handle on namespace.loadedUrls but not the internal loadedUrls right?